src/index.html

```diff
@@ -1,13 +1,60 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
         content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+    <title>Product cards</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <div class="product" data-qa="card">
+      <img
+        src="images/imac.jpeg"
+        alt="APPLE A1419 iMac 27' Retina 5K Monoblock (MNED2UA/A)"
+        class="product__image"
+      >
+      <h2 class="product__name">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
+      <div class="product__code">
+        <p>
+          Prodduct code: 195434
+        </p>
+      </div>
```

Comment:
div is redudant wrapper
```suggestion
      <p class="product__code">
        Prodduct code: 195434
      </p>
```